server/Tests/BudgetTracker.BudgetSquirrel.Application.Tests/UnitTests/Budgeting/BudgetCreationTests.cs

```diff
@@ -0,0 +1,18 @@
+using BudgetTracker.Business.Budgeting;
+using BudgetTracker.TestUtils.Budgeting;
+using System;
+using Xunit;
+
+namespace BudgetTracker.BudgetSquirrel.Application.Tests
+{
+    public class BudgetCreationTests
+    {
+        [Fact]
+        public void TestBudgetSavedWhenValidRequestReceived()
+        {
+            BudgetBuilder bb = new BudgetBuilderFactory<Budget>().GetBuilder() as BudgetBuilder;
+            Budget budget = bb.Build();
+            Console.WriteLine(budget.Duration.Id);

```

Comment:
Doesn't look like you are testing anything here?

Reply:
Oh yeah I gotta delete this. I was debugging the builders